Dockerfile

```diff
@@ -6,22 +6,34 @@ RUN git --version
 RUN clang-format -version
 RUN shellcheck --version
 
+# Create an user so we don't run this as root.
+ARG USERID
+ARG USERNAME
+ARG USERGRP
+
+RUN groupadd -g $USERGRP primarygroup
```

Comment:
If we are hardcoding `primarygroup` here we may as well hardcode the username, e.g. we may have `docker:docker` or something like that; in practice I don't think we rely on the username being consistent anyways.

Reply:
Done